debian/bionic/foreman/rules

```diff
@@ -33,9 +33,10 @@ build:
 	dh $@ --with=systemd
 
 override_dh_installinit:
-	dh_installinit
+	dh_installinit --only-scripts
 
 override_dh_systemd_enable:
-	dh_systemd_enable --package=foreman-service
+	dh_systemd_enable --name=foreman.service --package=foreman-service
+	dh_systemd_enable --name=foreman.socket --package=foreman-service
```

Comment:
So far in testing, this has the effect of not creating these services during `apt-get install foreman` and requires a user to install `foreman-service` which is in line with RPM. However, this still seems to start the services so working to figure out how to prevent that (going to try the --no-start).

Reply:
I think that may have been the true bug. We did do it right for the dynflow part, but omitted it for the actual service.